source/_components/switch.wake_on_lan.markdown

```diff
@@ -31,13 +31,29 @@ switch:
     mac_address: "00-01-02-03-04-05"
 ```
 
-Configuration variables:
-
-- **mac_address** (*Required*): MAC address to send the wake up command to.
-- **name** (*Optional*): The name of the switch. Default is 'Wake on LAN'.
-- **host** (*Optional*): The IP address or hostname to check the state of the device (on/off).
-- **turn_off** (*Optional*): Defines an [action](/getting-started/automation/) to run when the switch is turned off.
-- **broadcast_address** (*Optional*): The IP address of the host to send the magic packet to (default 255.255.255.255).
+{% configuration %}
+mac_address:
+  description: MAC address to send the wake up command to.
+  required: true
+  type: string
+name:
+  description: The name of the switch. Default is 'Wake on LAN'.
```

Comment:
What about adding the `default` key?

Reply:
:+1: 
And now remove it from the description?

Reply:
Done 👍